source/packages/modules.md

```diff
@@ -100,6 +102,93 @@ import {default as Value, a, F} from "./exporter";
+
+There is a case where you might *need* to use CommonJS, even if your project has the `ecmascript` package: if you want to conditionally include a module. `import` statements must be at top-level scope, so they cannot be within an `if` block. If you’re writing a common file, loaded on both client and server, you might want to import a module in only one or the other environment:
+
+```js
+if (Meteor.isClient) {
+  require('./client-only-file.js');
+}
+```
+
+Note that dynamic calls to `require()` (where the name being required can change at runtime) cannot be analyzed correctly and may result in broken client bundles. This is also discussed in [the guide](http://guide.meteor.com/structure.html#using-require).
+
+### CoffeeScript
+
+CoffeeScript has been a first-class supported language since Meteor’s early days. Even though today we recommend ES2015, we still intend to support CoffeeScript fully.
+
+CoffeeScript lacks support for `import` and `export`, though [the project maintainers are in the early stages of working to change that](https://github.com/jashkenas/coffeescript/issues/4078). In the meantime, CoffeeScript users can enjoy Meteor’s new modules support by using CommonJS syntax. If you use CoffeeScript’s [destructuring](http://coffeescript.org/#destructuring), the syntax is remarkably similar to the ES2015 examples you see above. For example, ES2015 `import` lines like these:
+
+```js
+import { AccountsTemplates } from 'meteor/useraccounts:core';
+import '../imports/startup/client/routes.js';
+```
+
+can be written in CoffeeScript using CommonJS `require` like this:
+
+```coffeescript
+{ AccountsTemplates } = require 'meteor/useraccounts:core'
+require '../imports/startup/client/routes.coffee'
+```
+
+(assuming you rename `routes.js` to `routes.coffee`). Note that files don’t need a `module.exports` if they’re required like `routes.coffee` is in this example, without assignment to any variable. The code in `routes.coffee` will simply be included and executed in place of the above `require` statement.
+
+ES2015 `export` statements like these:
+
+```js
+export const insert = new ValidatedMethod({ // ...
+export default incompleteCountDenormalizer;
+```
+
+can be rewritten to use CommonJS `module.exports`:
+
+```coffeescript
+module.exports.insert = new ValidatedMethod # ...
+module.exports = incompleteCountDenormalizer
+```
+
+You can also simply write `exports` instead of `module.exports` if you prefer.
+
+Note that [backticks](http://coffeescript.org/#embedded) do *[not](https://github.com/meteor/meteor/issues/6000)* work; your `.coffee` files will be transpiled into JavaScript, but not then handed off to the `ecmascript` package for further transpiling. Any JavaScript you type in backticks will be passed through unmodified all the way to the browser or Node.js runtime.
```

Comment:
> Note that backticks do not work;

maybe: Note that putting ecmascript inside backticks does not work

to make clear from the start you're not saying they don't work period.